packages/vscode-extension/src/debugging/DebugAdapter.ts

```diff
@@ -368,7 +380,10 @@ export class DebugAdapter extends DebugSession {
   }
 
   private cdpMessageId = 0;
-  private cdpMessagePromises: Map<number, (result: any) => void> = new Map();
+  private cdpMessagePromises: Map<
+    number,
+    { resolve: (result: any) => void; reject: (error: any) => void }
```

Comment:
Nit: extract to own type so it fits in one line and replace `any` with `unknown`.